eventauth/eventauth.go

```diff
@@ -251,6 +229,73 @@ func redactEventAllowed(event Event, authEvents AuthEvents) error {
 	panic("Not implemented")
 }
 
+// defaultEventAllowed checks whether the event is allowed by the default
+// checks for events.
+// It returns an error if the event is not allowed or if there was a
+// problem loading the auth events needed.
```

Comment:
Do we really want to bundle explicit event rejection into the same error response as "general problems loading the auth events"? I worry the latter may be transient, but the former isn't?